### PR TITLE
Fix canary build issues

### PR DESCRIPTION
**Pull request by travishathaway**
This pull request is a result of the work done in the following issue: https://github.com/conda/conda-build/issues/4572

Please see that issue for full details. Any suggestions to change this pull request should be made here and not that issue.

### CODE REVIEW

**Review by dholth — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
I've bumped the build number to `1`

**Comment by skupr-anaconda** [comment]
`conda init --all` causes the error. I've just checked it locally

**Comment by skupr-anaconda** [comment]
On `win-64` the Prefect builder issue:
`CondaValueError: prefix already exists: C:\Users\BUILDE~1\AppData\Local\Temp\built-conda-test-env`

**Comment by skupr-anaconda** [comment]
`s390x` agent is not working currently on Prefect

File: recipe/run_test.sh [hunk 1]
```diff
@@ -7,13 +7,24 @@ unset CONDA_SHLVL
 unset _CE_CONDA
 unset _CE_M
 unset CONDA_EXE
+
+# Used to rewrite the entry point script so it points to the correct python interpreter
+#conda init --all
```
Comment (jezdez) [inline recipe/run_test.sh:12]:
```suggestion
conda init --all
```

**Review by jezdez — COMMENTED** [review]
Let's try commenting this in again

**Comment by travishathaway** [comment]
This pull request is no longer necessary as the fixes for the `run_test.sh` file have been carried over to this pull request instead:
- https://github.com/AnacondaRecipes/conda-feedstock/pull/14

**Pull request closed without merging**